public/index.html

```diff
@@ -52,13 +52,81 @@
+    0)))}}var d=a;"undefined"!==typeof f?d=a[f]=[]:f="mixpanel";d.people=d.people||[];d.toString=function(b){var a="mixpanel";"mixpanel"!==f&&(a+="."+f);b||(a+=" (stub)");return a};d.people.toString=function(){return d.toString(1)+".people (stub)"};k="disable time_event track track_pageview track_links track_forms register register_once alias unregister identify name_tag set_config reset opt_in_tracking opt_out_tracking has_opted_in_tracking has_opted_out_tracking clear_opt_in_out_tracking people.set people.set_once people.unset people.increment people.append people.union people.track_charge people.clear_charges people.delete_user".split(" ");
+    for(h=0;h<k.length;h++)e(d,k[h]);a._i.push([b,c,f])};a.__SV=1.2;b=e.createElement("script");b.type="text/javascript";b.async=true;b.src="undefined"!==typeof MIXPANEL_CUSTOM_LIB_URL?MIXPANEL_CUSTOM_LIB_URL:"file:"===e.location.protocol&&"//cdn4.mxpnl.com/libs/mixpanel-2-latest.min.js".match(/^\/\//)?"https://cdn4.mxpnl.com/libs/mixpanel-2-latest.min.js":"//cdn4.mxpnl.com/libs/mixpanel-2-latest.min.js";c=e.getElementsByTagName("script")[0];c.parentNode.insertBefore(b,c)}})(document,window.mixpanel||[]);
+  </script><!-- end Mixpanel -->
+
+  <!-- 
+    Unlock snippet scripts modified from 
+    https://github.com/unlock-protocol/react-example/blob/master/public/index.html
+  -->
+  <script>
+    // set network based on start scripts environment variables
+    const nodeEnv = "%NODE_ENV%";
+    const ethEnv = "%REACT_APP_ENV%";
+    const networkId = (ethEnv.includes("REACT_APP_ENV") && nodeEnv === "production") ? 1 
+    : (ethEnv.includes("REACT_APP_ENV") && nodeEnv !== "production") ? 4  
+    : ethEnv === "rinkeby" ? 4 
+    // : ethEnv === "kovan" ? 42 // include kovan when supported by unlock paywall
+    : ethEnv === "live" ? 1
+    : 4;
+
+    (function (d, s) {
+      // if Rinkeby set, use Rinkeby paywall, else use Mainnet
+      let testnetUrl = networkId === 4 ? 'staging-' : '';
+      let js = d.createElement(s);
+      let sc = d.getElementsByTagName(s)[0];
+        js.src = `https://${testnetUrl}paywall.unlock-protocol.com/static/unlock.1.0.min.js`;
+        sc.parentNode.insertBefore(js, sc);
+    }(document, "script"));
+
+    /* 
+      Kovan is included, but currently only Rinkeby and Mainnet is supported with paywall
+      tokens entered first to last are displayed right to left in paywall modal
+      Kovan lock is included for when kovan is supported by Unlock paywall
+      NOTE: Replace Mainnet keys with keys Kickback owns
+    */
+    const locks = {
```

Comment:
Is there any way to abstract these into the main `src` directory. It's not very visible having the locks in the index.html.

Reply:
I don't think so. I looked through Unlock's documentation, and looked at the React example in Unlock's Github. They had this config in `public/index.html`.  I also don't think there's a way to import a file from `src` to `public`, else I'd be using `env.json` instead of passing environment variables through `package.json`. Unless @julien51 has any suggestions.

Reply:
Actually... I think this just adds a variable to `window`. Let me test something

Reply:
Ok. Locks moved to `env.json`. Readme updated as well. 393de51